fmriprep/workflows/fieldmap/se_pair_workflow.py

```diff
@@ -101,41 +97,6 @@ def se_pair_workflow(name='Fieldmap_SEs', settings=None):  # pylint: disable=R09
 
     return workflow
 
-def fieldmap_to_phasediff(name='Fieldmap2Phasediff'):
```

Comment:
I'm not totally convinced that we need to have fieldmap submodules, what is the rationale behind this?